### PR TITLE
Feature/comparison card

### LINKED ISSUE #26: check type 'string' on handleComparisonClick
As a developer, 
When I pass a typed parameter to the function `handleComparisonClick`, 
I want to see that the string being passed reflecting ONLY the `ComparisonCategory` not just any string,
So that I know my values are utilizing correct TypeScript properties.

### LINKED ISSUE #38: create ComparisonCard component
As a user, 
When I view the comparison data, 
I want to see cards that display the data,
So that I can read and view the data easily.

### PR DESCRIPTION
**Pull request by alyssajoyner**
## What's this PR do?
- Create `ComparisonCard` component
- Create logic and boiler plate text for comparison details page
- Add initial styling to comparison details page
- Add additional state in `ComparisonDetails` to account for both displays of icons

## Where should the reviewer start?
- `ComparisonDetail.tsx` and `ComparisonCard.tsx`

## How should this be manually tested?
- Pick a comparison from the drop down menu and click the button
- Ensure the display shows a different number of 'Xs' for each display
- NOTE: each 'X' represents 100 deaths

## Any background context you want to provide?
- The 'Xs' will be changed to icons on the next styling branch

## Screenshots/GIFs
Screen Shot 2021-01-12 at 11 52 30 AM

### CODE REVIEW

File: src/ComparisonCard/ComparisonCard.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.cards {
+  border: solid black 1px;
+  padding: 20px;
+  margin: 35px;
+  height: 300px;
+  width: 250px;
```
Comment (kncaputo) [inline src/ComparisonCard/ComparisonCard.scss:6]: I added these in pxs but we should switch these to ems

File: src/ComparisonCard/ComparisonCard.tsx [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+const ComparisonCard = (prop: {title?: string, body?: JSX.Element[], text?: JSX.Element, styleId: string, image?: string}) => {
   const generateComparisonCards = () => {
-    
+    return(
+      <section className={`${prop.styleId}-cards cards`}>
+        <section className='card-title'>
+          <p>{prop.title}</p>
+        </section>
+        <section>{prop.text}</section>
+        <section className={`${prop.styleId}-body`}>{prop.body}</section>
```
Comment (kncaputo) [inline src/ComparisonCard/ComparisonCard.tsx:11]: We need to add the source next

**Review by kncaputo — APPROVED** [review]
Looking good. I want to add the source and the icon to the card next.

File: src/comparisonData.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ export type ComparisonData = {
 
 export interface Comparison {
   id: number,
-  category: ComparisonCategory | undefined,
+  category: ComparisonCategory | string,
```
Comment (Codo-Baggins) [inline src/comparisonData.tsx:10]: Nice catch. I think undefined would not have been desirable here.

File: src/App/App.tsx [hunk 4]
```diff
@@ -51,7 +51,7 @@ class App extends Component<Props, State> {
       },
       selectedComparison: {
         id: 0,
-        category: undefined,
+        category: '',
```
Comment (Codo-Baggins) [inline src/App/App.tsx:55]: Good thought process in changing this to an empty string.

**Pull request merged**